torch/linalg/__init__.py

```diff
@@ -12,55 +12,63 @@
 cholesky = _add_docstr(_linalg.linalg_cholesky, r"""
 linalg.cholesky(input, *, out=None) -> Tensor
 
-Computes the Cholesky decomposition of a Hermitian (or symmetric for real-valued matrices)
-positive-definite matrix or the Cholesky decompositions for a batch of such matrices.
-Each decomposition has the form:
+Computes the Cholesky decomposition of a complex Hermitian (or real symmetric) positive-definite matrix or batch of matrices.
+
+For a complex Hermitian (or real symmetric) matrix :math:`A`, this is defined as
 
 .. math::
 
-    \text{input} = LL^H
+    A = LL^{\text{H}}
+
+where :math:`L` is a lower triangular matrix and
+:math:`L^{\text{H}}` denotes the conjugate transpose in the complex case and the transpose in the real case.
```

Comment:
"denotes" -> "is"

"the conjugate transpose of L when L is complex, and the transpose of L when L is real-valued."

Reply:
Corrected throughout the docs.

Reply:
After seeing the final version, I changed it for:
>" the conjugate transpose when L is complex, and the transpose when L is real-valued."

I think it is more readable as it avoids having to mention the matrix 5 times in the same sentence.